app/views/catalog/_home_text.html.erb

```diff
@@ -40,13 +40,10 @@
       <p>Choose a library to begin:</p>
       <ul class="library-list">
         <% @display_libs = {} %>
-        <%- @libraries.each do |key, value| %>
-          <% @display_libs[value["name"]] = key unless key == "free" || key == "neos" %>
-        <% end %>
-        <% @display_libs.sort.each do |value, key| %>
+        <% neos_libraries.each do |library| %>
           <li>
-            <%= link_to image_tag("library-images/"+key+".jpg"), "/?lib="+key %>
-            <%= link_to value, "/?lib="+key %>
+            <%= link_to image_tag("library-images/"+library.short_code+".jpg"), "/?lib="+library.short_code %>
```

Comment:
maybe string interpolate here instead of concatation
```suggestion
            <%= link_to image_tag("library-images/#{library.short_code}.jpg"), "/?lib=#{library.short_code}" %>
```